src/services/sessionInvite.js

```diff
@@ -438,6 +620,30 @@ module.exports = class UserInviteHelper {
 			)
 
 			await this.fetchMentorIds(sessionCreationOutput)
+
+			const defaultOrgId = await getDefaultOrgId()
+			if (!defaultOrgId)
+				return responses.failureResponse({
+					message: 'DEFAULT_ORG_ID_NOT_SET',
+					statusCode: httpStatusCode.bad_request,
+					responseCode: 'CLIENT_ERROR',
+				})
+			const sessionModelName = await sessionQueries.getModelName()
+
+			let entityTypes = await entityTypeQueries.findUserEntityTypesAndEntities({
```

Comment:
Pull this out of the functions, since we're using the same data in multiple places.